packages/flutter/lib/src/widgets/transitions.dart

```diff
@@ -145,6 +195,7 @@ class FadeTransition extends AnimatedComponent {
   }
 }
 
+/// Animates the background color of a widget.
 class ColorTransition extends AnimatedComponent {
```

Comment:
Looks like it was a primitive version of ModalBarrier from the fn2 days.  I'll post a separate patch to remove it.